string name   #Name of path
---
bool success  #Error flag
